tslint.json

```diff
@@ -1,55 +0,0 @@
-{
```

Comment:
How about having a tslint like
```
{
    "extends": "./node_modules/resin-lint/config/tslint-prettier.json"
}
```
So that IDE's can easily find the configuration?

Reply:
Done